x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -609,7 +641,7 @@ export const ConfigurationStepForm: FC<ConfigurationStepProps> = ({
               placeholder={i18n.translate(
                 'xpack.ml.dataframe.analytics.create.dependentVariablePlaceholder',
                 {
-                  defaultMessage: 'dependent variable',
+                  defaultMessage: 'Select the field to be used as the dependent variable.',
```

Comment:
>                  defaultMessage: 'Select the field to be used as the dependent variable.',

>    defaultMessage: 'The field that you want to predict the value of.',

I think that the list of value that you can choose from is filtered (for regression, the docs say it has to be a numeric field; for classification, the docs say the field must be numeric (integer, short, long, byte), categorical (ip or keyword), or boolean). Would it be useful (and feasible) to have that context-specific detail? For example:

"Select the numeric field that you want to predict"